generate-docker-image-tags/action.yml

```
@@ -57,7 +57,9 @@ runs:
         then
           TAGS="latest,$BRANCH,$COMMIT_SHA"
         else
-          TAGS="${{ inputs.prefix }}.$BRANCH,${{ inputs.prefix }}.$COMMIT_SHA"
+          BADCHAR="/"
+          GOODCHAR="-"
+          TAGS="${{ inputs.prefix }}.${BRANCH//$BADCHAR/$GOODCHAR},${{ inputs.prefix }}.$COMMIT_SHA"
```

I was trying to test this locally and found that although `${var////-}` doesn't work in zsh (but it does in bash).

I'm happy to keep what you have here but I think the following would be more readable:
```suggestion
          SLASHLESS_BRANCH=`echo $BRANCH | tr / -`
          TAGS="${{ inputs.prefix }}.$SLASHLESS_BRANCH,${{ inputs.prefix }}.$COMMIT_SHA"
```